### ISSUE TITLE
compounds without formula specified (only m/z) are not found in library search

### ISSUE THREAD

**Issue by chubukov** [ElucidataInc/ElMaven#1282]
In v0.10, if you have no formula specified for the compound, it doesn't get picked up when doing compound database search. Guessing maybe this happened with the adduct-oriented update? Fairly major issue for us.

**Comment by saifulbkhan** [comment]
@chubukov You are right. I just checked. This was an issue in the adduct branch but I clearly remember dealing with it. But that branch was also rebased many times, and this must have crept back in somehow.

We will fix this ASAP. I will add links to the installers here (forked from v0.10.0 master), if that helps.

**Comment by saifulbkhan** [comment]
@chubukov For what its worth, you can now specify neutral masses in the compound database as well - under a column named "mass". This mass will be adjusted according to the current global ionization polarity.

**Comment by chubukov** [comment]
@saifulbkhan that seems like a good workaround, thanks. It ran, but I got some other strange behaviors -- I got the nan issue again (#1259) and also, the compound names did not get populated

**Comment by chubukov** [comment]
@saifulbkhan that's a good workaround, thanks. That seemed to mostly work for me -- one issue I see is I got nan again for a lot of observedMz values, similar to #1259

**Comment by saifulbkhan** [comment]
@chubukov I see. We are looking into the nan thing. Have not been able to reproduce it until now. I am afraid it could be a framework-level bug, but we will resolve it, one way or another.

Also, adding links to installers with the compounds-with-mz fix, for everyone else who prefers to use raw m/z values:
Windows: https://elmaven-installers.s3-us-west-2.amazonaws.com/Windows/El-MAVEN-fix_mz_detection.exe
Mac OS: https://elmaven-installers.s3-us-west-2.amazonaws.com/Mac/El-MAVEN-fix_mz_detection.zip

This fix will be available with the next (v0.11.0) release.
